src/components/Transactions/TransactionReceive.tsx

```diff
@@ -166,9 +166,11 @@ export const TransactionReceive = ({
               <Link
                 isExternal
                 color='blue.500'
-                href={`${txDetails.explorerAddressLink}${txDetails.to ?? txDetails.from}`}
+                href={`${txDetails.explorerAddressLink}${txDetails.ensFrom || txDetails.from}`}
```

Comment:
look at all the bikes we need to fit in that shed
```suggestion
                href={`${txDetails.explorerAddressLink}${txDetails.ensFrom ?? txDetails.from}`}
```